contents/docs/session-replay/troubleshooting.mdx

```diff
@@ -4,6 +4,18 @@ title: Troubleshooting and FAQs
 
 Having trouble with recordings? Below are some tips for getting past some common issues
 
+The most common solution is...
+
+#### Update posthog-js
+
+We're always making improvements to the recordings feature, so you'll want to make sure that you're running the latest version of posthog-js on your website.
+
+To check the version that you're using, you can run the following in your console:
```

Comment:
```suggestion
To check the version that you're using, you can run the following in your browser console:
```
I'm not sure this is right, so if it can be run in other consoles, feel free to ignore.